config_dev.env.example

```diff
@@ -24,10 +24,7 @@
 DEBUG=True
 
 # Languages that this smbackend instance will support. This affects
-# several things and cannot be changed nilly-willy. You will also need
-# to rebuild your elasticsearch indexes: `manage.py rebuild_indexes`,
-# but only if you are actually running elasticsearch, ie. specifying
-# ELASTICSEARCH_URL
+# several things and cannot be changed nilly-willy. 
```

Comment:
'willy-nilly'

Reply:
Fixed, but that nilly-willy expression has been there before I made my editions. Maybe the previous editor had some meaning to nilly-willy.